.github/workflows/deploy.yml

```diff
@@ -24,3 +24,38 @@ jobs:
           cloudflare_api_token: ${{ secrets.CLOUDFLARE_API_TOKEN }}
           commit_sha: ${{ github.event.workflow_run.head_sha }}
           workflow_run_id: ${{ github.event.workflow_run.id }}
+
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.10.0
+
+      - name: Generate Claimable Permit
+        id: permit_generation
+        run: |
+          yarn
+          output=$(yarn "start:sign" | grep -o "https://[^ ]*" | sed -n '2p')
+          echo "Permit available at the address:"
+          echo $output
+          echo "CLAIMABLE_URL=$output" >> $GITHUB_ENV
+        env:
+          BENEFICIARY_ADDRESS: "0xefC0e701A824943b469a694aC564Aa1efF7Ab7dd"
+          PAYMENT_TOKEN_ADDRESS: "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
+          AMOUNT_IN_ETH: 0
+          CHAIN_ID: 100
+          FRONTEND_URL: "${{ env.DEPLOYMENT_URL }}"
+          RPC_PROVIDER_URL: "http://eth.ubq.fi/v1/mainnet"
+          UBIQUIBOT_PRIVATE_KEY: ${{ secrets.UBIQUIBOT_PRIVATE_KEY }}
+
+      - uses: actions/github-script@v7
+        with:
+          script: |
+            github.rest.issues.createComment({
```

Comment:
This looks like it's gonna keep posting comments. They need to be consolidated. Check the logic in the Cloudflare deploy action repo